src/main/java/org/spongepowered/common/config/category/TimingsCategory.java

```diff
@@ -33,22 +33,22 @@
 @ConfigSerializable
 public class TimingsCategory extends ConfigCategory {
 
-    @Setting
+    @Setting(comment = "???")
     private boolean verbose = false;
 
-    @Setting
+    @Setting(comment = "???")
     private boolean enabled = true;
 
-    @Setting("server-name-privacy")
+    @Setting(value = "server-name-privacy", comment = "???")
```

Comment:
Hides the server name in the Airkar webviewer to no leak the l33t server you are running